tsconfig.json

```diff
@@ -11,7 +11,7 @@
 		"forceConsistentCasingInFileNames": true,
 		"isolatedModules": true,
 		"jsx": "preserve",
-		"lib": ["dom", "es2019"],
+		"lib": ["dom", "dom.iterable", "esnext"],
```

Comment:
This is just an update to TypeScript so it's aware of the `.replaceAll` used in the `stripEmojis` method.